Assets/Editor/MoNA/SendToPlayground.cs

```diff
@@ -0,0 +1,30 @@
+
+using UnityEngine;
+using UnityEditor;
+using UnityToolbarExtender;
+
+namespace MoNA
+{
+    [InitializeOnLoad]
+    public class SendToPlayground
+    {
+        static SendToPlayground()
+        {
+            ToolbarExtender.RightToolbarGUI.Add(OnToolbarGUI);
+        }
+
+        static void OnToolbarGUI()
+        {
+            GUILayout.FlexibleSpace();
+
+            GUI.skin.button.fontSize = 13;
+            if (GUILayout.Button(new GUIContent("▶️ Playground", "Build and open playground")))
+            {
+                Helpers.UpsertExportsDirectory();
+                BuildPipeline.BuildAssetBundles(Constants.PlaygroundDirectory, BuildAssetBundleOptions.None, BuildTarget.WebGL);
+                Helpers.OpenDirectory(Constants.ExportsDirectory);
+                Application.OpenURL("https://www.mona.gallery/playground");
```

Comment:
Could you create a reference to this in the `Constants` - this URL may be subject to change so just easier to track down that way :)

Reply:
Got it, changed it + formating